### PR TITLE
chore(DIA-64): Update curated collections in nav

### PR DESCRIPTION
**Pull request by jo-rs**
The type of this PR is: **Chore**

This PR solves [DIA-64]

### Description
Updates in curated collections in nav
Screenshot 2023-08-18 at 15 22 17
Screenshot 2023-08-18 at 15 22 08

[DIA-64]: https://artsyproduct.atlassian.net/browse/DIA-64?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/NavBar/menuData.ts [hunk 1]
```diff
@@ -137,6 +92,14 @@ export const ARTISTS_SUBMENU_DATA: MenuLinkData = {
...
         },
       },
+      // Empty column
+      {
+        text: "",
+        menu: {
+          title: "",
+          links: [],
+        },
+      },
```
Comment (jo-rs) [inline src/Components/NavBar/menuData.ts:102]: Open to other suggestions here

Comment (jonallured) [reply src/Components/NavBar/menuData.ts:102]: 🤷 seems legit

**Review by jonallured — APPROVED** [review]

File: src/Components/NavBar/menuData.ts [hunk 2]
```diff
@@ -37,70 +37,25 @@ export const ARTISTS_SUBMENU_DATA: MenuLinkData = {
...
-            {
-              text: "Rising Artists of Asia & Its Diaspora",
-              href: "/collection/rising-artists-of-asia-and-its-diaspora",
+              text: "Curators’ Picks: Emerging",
+              href: "/collection/curators-picks-emerging",
             },
             {
-              text: "Women Artists Now",
-              href: "/collection/women-artists-to-watch",
+              text: "Curators’ Picks: Blue-Chip",
```
Comment (araujobarret) [inline src/Components/NavBar/menuData.ts:52]: is it like this or `Curator's`?
Line 43 is in this form and here it's in a different form

Comment (jo-rs) [reply src/Components/NavBar/menuData.ts:52]: Good question 🤔  
It's like that on the ticket, but maybe @jpoczik can confirm?
It's a bit odd that the main header would refer to an individual curator, but the sub links to the collective. But, I'm not a native speaker 🤷

Comment (jonallured) [reply src/Components/NavBar/menuData.ts:52]: I believe it should match what you see when you go to the page:

https://www.artsy.net/collection/curators-picks-blue-chip?metric=in

Which looks like this:

```
Curators’ Picks: Blue-Chip
```

Comment (jpoczik) [reply src/Components/NavBar/menuData.ts:52]: yes confirming what jon says that it's from multiple curators so **Curators’ Picks: Blue-Chip** is correct

**Pull request merged**